vpr/src/device/rr_graph_builder.h

```diff
@@ -235,6 +280,21 @@ class RRGraphBuilder {
     /* Fast look-up for rr nodes */
     RRSpatialLookup node_lookup_;
 
+    /* Segment relatex data
+     * Segment info should be corrected annotated for each rr_node
+     * whose type is CHANX and CHANY
+     */
+    vtr::vector<RRSegmentId, t_segment_inf> rr_segments_; /* detailed information about the segments, which are used in the RRGraph */
```

Comment:
Should add type definition comments here. Or add comments pointing to the flyweight data structure ``t_segment_inf``.